src/main/java/cs361/battleships/models/Sub.java

```diff
@@ -0,0 +1,25 @@
+package cs361.battleships.models;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.google.common.collect.Sets;
+import com.mchange.v1.util.CollectionUtils;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import java.util.Set;
+
+public class Sub extends Ship {
+    private String name;
+    private int size;
+    public Sub(){
+        this.name = "SUB";
```

Comment:
Whoops you need to change the size to 5 instead of 3, simple copy-pasta error :)